projects/pastanaga-angular/src/lib/controls/textfield/select/select.component.ts

```diff
@@ -60,6 +60,8 @@ export class SelectComponent extends PaFormControlDirective implements OnChanges
         return this._dim;
     }
 
+    @Input() optionsPosition?: PositionStyle;
```

Comment:
another way to do it could maybe be creating a directive that would add this dynamically in it's children.
don't know how this is would be simple but it would avoid adding the same input to any component that use a popup... it would also allow to reuse it anytime we face this issue...